### PR TITLE
Fix- Defintions of domain name page is modified

### PR DESCRIPTION
**Pull request by Harishn97**

### CODE REVIEW

**Comment by Harishn97** [comment]
<html lang="en">
<head>
  <title>How the Internet Works</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet"
href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
    <link rel="stylesheet" href="css/index.css" type="text/css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
    <link rel="stylesheet" href="css/index.css">
</head>
<body background="images/background.jpg" >

<nav class="navbar navbar-inverse">
  <div class="container-fluid">
    <div class="navbar-header">git
      <button type="button" class="navbar-toggle"
data-toggle="collapse" data-target="#myNavbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="#">How the Internet Works</a>
    </div>
    <div class="collapse navbar-collapse navbar-right " id="myNavbar">
      <ul class="nav navbar-nav">
        <li class="active"><a href="index.html">Home</a></li>
        <li><a href="#">About</a></li>
        <li><a href="#">Team</a></li>

      </ul>
    </div>
  </div>
</nav>

<div class="container">
  <div class="row">
    <div class="col-md-4">
      <a href="Domain_names_and_ICANN.html" style="color:
#1a1e21;text-decoration: none;">
        <div class="card card-1">
        <h3>Domain Names and ICANN</h3>
        <p>A domain name is a string of text that maps to a numeric
Internet Protocol address, used to access a website from client
software</p>
      </div></a>
    </div>
    <div class="col-md-4">
      <a href="IP_Address_Packets_Routing.html" style="color:
#1a1e21;text-decoration: none;">
        <div class="card card-2">
        <h3>IP Addresses, Packets and Routing</h3>
        <p>IP address is a numerical label assigned to each device
connected to a network that uses the IP for communication</p>
      </div></a>
    </div>
    <div class="col-md-4">
      <a href="DNS.html" style="color: #1a1e21;text-decoration: none;">
        <div class="card card-3">
        <h3>DNS- Domain Name System</h3>
        <p>The Domain Name System (DNS) is the phonebook of the
Internet. Humans access information online through domain names.</p>
      </div></a>
    </div>
  </div><br>
      <div class="row" >
      <div class="col-md-4">
      <a href="protocols.html" style="color: #1a1e21;text-decoration: none;">
        <div class="card card-4">
        <h3>HTTP and HTTPS protocols</h3>
        <p>The Hypertext Transfer Protocol (HTTP) is an application
layer protocol for distributed, collaborative, hypermedia information
systems</p>
      </div></a>
    </div>
      <div class="col-md-4">
      <a href="W3C_HTML_CSS.html" style="color: #1a1e21;text-decoration: none;">
        <div class="card card-5">
        <h3>W3C and HTML and CSS</h3>
        <p>HTML is the markup language for designed to be displayed in
a web browser. CSS is a style sheet language used of a document
written in HTML</p>
      </div></a>
      </div>
    </div>
</div>
</body>
</html>
